### DISCUSSION TITLE
Phase-field crystal in MOOSE

### DISCUSSION THREAD

**Discussion by vpavan1920** [idaholab/moose#21333] [Q&A Modules: Phase field] [answered]
Is there a publication that discusses the implementation and PFC formulation in MOOSE ?

**Comment by GiudGiud** [comment, marked as answer]
@laagesen @SudiptaBiswas @jiangwen84

**Reply by SudiptaBiswas** [reply]
We don't have phase-field crystal model in MOOSE.

**Reply by laagesen** [reply]
Actually we do have some PFC capability, although to be honest it has not seen much development in recent years. There is not a publication, but there is some documentation here:
https://mooseframework.inl.gov/moose/source/kernels/CHBulkPFCTrad.html
Basically it implements a PFC free energy that can be used in a Cahn-Hilliard equation. There are some example input files and links to other classes on the documentation page.

**Reply by vpavan1920** [reply]
Thanks, I'll look at the documentation accordingly
